app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::UsersController < ActionController::API
+  def ban
+    return render status: :precondition_failed, json: I18n.t('users.invalid_cpf') unless CPF.valid?(cpf_param)
+
+    client = Client.find_by(cpf: cpf_param)
+    personal = Personal.find_by(cpf: cpf_param)
+    return render status: :not_found, json: I18n.t('users.no_account') if client.nil? && personal.nil?
```

Comment:
```suggestion
    return render status: :not_found, json: I18n.t('users.no_account') unless client || personal
```

Parece uma mudança boba mas pensando na leitura acho que fica legal, o que acham?

Reply:
Eu sou muito burro pra entender unless, mas parece boa mesmo.